pkg/api/handlers/libpod/manifests.go

```diff
@@ -162,13 +162,36 @@ func ManifestAdd(w http.ResponseWriter, r *http.Request) {
 	// Wrapper to support 3.x with 4.x libpod
 	query := struct {
 		entities.ManifestAddOptions
-		Images []string
+		Images    []string
+		TLSVerify bool `schema:"tlsVerify"`
 	}{}
 	if err := json.NewDecoder(r.Body).Decode(&query); err != nil {
 		utils.Error(w, http.StatusInternalServerError, errors.Wrap(err, "Decode()"))
 		return
 	}
 
+	authconf, authfile, err := auth.GetCredentials(r)
+	if err != nil {
+		utils.Error(w, http.StatusBadRequest, errors.Wrapf(err, "failed to parse registry header for %s", r.URL.String()))
+		return
+	}
+	defer auth.RemoveAuthfile(authfile)
+	var username, password string
+	if authconf != nil {
+		username = authconf.Username
+		password = authconf.Password
+	}
+	query.ManifestAddOptions.Authfile = authfile
+	query.ManifestAddOptions.Username = username
+	query.ManifestAddOptions.Password = password
+	query.ManifestAddOptions.All = query.All
```

Comment:
This seems unrelated to the description of the PR. Is it intentional?

Reply:
@mtrmac Sorry for the stupid question, did you mean specifically the last line here `	query.ManifestAddOptions.All = query.All` or all of it
```
	query.ManifestAddOptions.Authfile = authfile
	query.ManifestAddOptions.Username = username
	query.ManifestAddOptions.Password = password
	query.ManifestAddOptions.All = query.All
```

Reply:
I'm sorry, I meant just the `.All` line.

(I’m not saying it’s incorrect, or unwanted, I haven’t checked; it just seems unrelated and I can‘t see any mention of that part.)

Reply:
It seems unnecessary. I removed it and tests are passing. I was working through getting credentials passed through properly and wasn't sure based on looking at the ImagePushOptions for Push whether this should also be added.